go/vt/vttablet/tabletserver/throttle/throttler.go

```diff
@@ -851,6 +852,9 @@ func (throttler *Throttler) refreshMySQLInventory(ctx context.Context) error {
 				return
 			}
 			if !throttler.isLeader.Load() {
+				// This tablet may have used to being the primary. Ensure the probes for this cluster are
+				// overwritten with an empty list
+				throttler.mysqlClusterProbesChan <- clusterProbes
```

Comment:
Is it possible to be more explicit here by sending nil, or at least `&mysql.ClusterProbes{}`?

Reply:
`&mysql.ClusterProbes{}` would be incorrect to send, because it must indicate the cluster name. The value sent here is initialized and correct.

Reply:
I know that it is correct, from looking above, but I think we should be explicit here to avoid surprises when anything else changes. Maybe we can explicitly set the probes to nil?
```
clusterProbes.InstanceProbes = nil 
throttler.mysqlClusterProbesChan <- clusterProbes
```

If it's a problem then it's fine. It just feels wrong to send whatever was initiated elsewhere w/o explicitly ensuring it's what we want/expect at that point.

Reply:
@mattlord I added comments to clarify what it is we do and why. I feel like the current code is the clearest and most correct because:

> whatever was initiated elsewhere

it was initiated 5 lines above as the default and empty probe result, before being populated. It is almost the minimal set of values. If we wanted to push a new one, we'd push pretty much the exact same thing.